fs/rc/jobs/job.go

```diff
@@ -406,3 +406,34 @@ func rcJobStop(ctx context.Context, in rc.Params) (out rc.Params, err error) {
 	job.Stop()
 	return out, nil
 }
+
+func init() {
+	rc.Add(rc.Call{
+		Path:  "job/stopgroup",
+		Fn:    rcGroupStop,
+		Title: "Stop all running jobs in a group",
+		Help: `Parameters:
+
+- group - name of the group (string).
+`,
+	})
+}
+
+// Stops all running jobs in a group
+func rcGroupStop(ctx context.Context, in rc.Params) (out rc.Params, err error) {
+	group, err := in.GetString("group")
+	if err != nil {
+		return nil, err
+	}
+	running.mu.RLock()
+	defer running.mu.RUnlock()
+	for _, job := range running.jobs {
+		if job.Group == group {
+			job.mu.Lock()
```

Comment:
Is taking this mutex necessary? We don't take it in rcJobStop?

Reply:
Thank you for the review. I believe we take it here in rcJobStop:
https://github.com/rclone/rclone/blob/ea2e951f5869e1a0eb6046e897b437f2e5b40126/fs/rc/jobs/job.go#L399
https://github.com/rclone/rclone/blob/ea2e951f5869e1a0eb6046e897b437f2e5b40126/fs/rc/jobs/job.go#L171-L176